### ISSUE TITLE
 mpclient port to Windows [Stand-Alone] Windows Console

### ISSUE THREAD

**Issue by TheM4rc0** [taviso/loadlibrary#72]
Hello, can this project be portened on a win32 system?

I would like to run the mpclient application on a windows system consol.

Can I build the mpclient together in Visual Studio?

thx

**Comment by lengocthuong15** [comment]
Yeah you can. Just create a new project as normal, then add file "mpclient.c" as your main source file. Next, add all files in loadlibrary/include as your header files. Adjust the configuration and code if needed
